### PR TITLE
Moving to 2018 Cosmic WFs

### PR DESCRIPTION
**Pull request by boudoul**
Updating runthematrix cosmic WFs with the latest and greatest 2018 Cosmic GTs
The corresponding WFs are 7.2 , 7.3, 7.4
This PR should be tested with #25131 - Can only be merged after it (this PR is using a GT defined there)

Thanks @lpernie for having provided the GTs

@mmusich @tsusa FYI

### CODE REVIEW

**Comment by prebello** [comment]
please test workflow 7.2,7.3,7.4 with #25131

**Comment by prebello** [comment]
+1

**Comment by boudoul** [comment]
Hi @prebello , If I can make a general suggestion for Pull Request review , please always wait for the end of tests , including comparison before signing - Looking at the output of the tests, comparisons and even logs can spot a lot of issues before the actual integration  - Thanks

**Comment by boudoul** [comment]
Hi @prebello , the comparison is still ongoing

**Comment by boudoul** [comment]
but let me also thank you for the prompt reaction looking at my  PRs ! :-)

**Comment by fabiocos** [comment]
+1

File: Configuration/PyReleaseValidation/python/relval_standard.py [hunk 1]
```diff
@@ -479,9 +479,9 @@
 
 workflows[7]  = ['', ['Cosmics','DIGICOS','RECOCOS','ALCACOS','HARVESTCOS']]
 workflows[7.1]= ['', ['CosmicsSPLoose','DIGICOS','RECOCOS','ALCACOS','HARVESTCOS']]
-workflows[7.2] = ['', ['Cosmics_UP17','DIGICOS_UP17','RECOCOS_UP17','ALCACOS_UP17','HARVESTCOS_UP17']]
-workflows[7.3] = ['', ['CosmicsSPLoose_UP17','DIGICOS_UP17','RECOCOS_UP17','ALCACOS_UP17','HARVESTCOS_UP17']]
-workflows[7.4] = ['', ['Cosmics_UP17','DIGICOSPEAK_UP17','RECOCOSPEAK_UP17','ALCACOS_UP17','HARVESTCOS_UP17']]
+workflows[7.2] = ['', ['Cosmics_UP18','DIGICOS_UP18','RECOCOS_UP18','ALCACOS_UP18','HARVESTCOS_UP18']]
+workflows[7.3] = ['', ['CosmicsSPLoose_UP18','DIGICOS_UP18','RECOCOS_UP18','ALCACOS_UP18','HARVESTCOS_UP18']]
+workflows[7.4] = ['', ['Cosmics_UP18','DIGICOSPEAK_UP18','RECOCOSPEAK_UP18','ALCACOS_UP18','HARVESTCOS_UP18']]
```
Comment (slava77) [inline Configuration/PyReleaseValidation/python/relval_standard.py]: this update disables tests of the 2017 cosmics.
Is it obvious that we will never need to reprocess any 2017 cosmic data?
If not, please re-add the 2017.

Comment (fabiocos) [reply Configuration/PyReleaseValidation/python/relval_standard.py]: @slava77 @boudoul what has been the policy so far? Of course we can easily restore the cosmics workflows for 2017, I approved the change as I saw just the last year for these tests. How far in the past would you like to go?

Comment (boudoul) [reply Configuration/PyReleaseValidation/python/relval_standard.py]: Hello @slava77 and @fabiocos , I'm happy to have one 2016 and 2017 , this makes sense - The last suggestion shounds very good to me ('Clone 7.2 for 2017 and 2016 as say 7.12 and 7.22?') I'm happy to make that PR by tomorrow (today is already full ...)

Comment (slava77) [reply Configuration/PyReleaseValidation/python/relval_standard.py]: I'm not particularly attached to either, but 7.3 (SP is for super-pointing, IIRC) may be a better choice to have something going through the tracker, not just through the muon system

Comment (fabiocos) [reply Configuration/PyReleaseValidation/python/relval_standard.py]: @slava77 @boudoul a 2017 workflow can be easily restored, I have a branch with it. But for a 2016 cosmics workflow which conditions should be used? @tocheng @lpernie could you please clarify

Comment (boudoul) [reply Configuration/PyReleaseValidation/python/relval_standard.py]: Thanks @fabiocos , sorry I couldnt make it yesterday, glad that you are looking at it 
- Answering the previous comment from @slava77 : 7.3 is SP , which means that it contains only cosmic tracks going through the pixel volume, while 7.2 is actually also filtering but going through the full tracker volume (strip + pixel) - If we want to have a relevant test of cosmic tracking etc , I would recommend to 'clone' 7.2 for 2017, 2016  . 
I let @tocheng @lpernie  answer you question for 2016 .
Thanks again.

**Pull request merged**